server/scripts/redirector.py

```diff
@@ -407,15 +422,21 @@ def redir_apidartdev(handler, *args, **kwargs):
         defaults={'_uri': '/be'}),
```

Comment:
Do double redirections work here?  Would it be better to redirect all to /main?

Reply:
Clearly, multiple redirections work, they are used already in the file, so OK.

Reply:
Yep, it sends back the redirects to the client one at a time and doesn't hit the browser redirect limit at all